### DISCUSSION TITLE
zooming in with drone camera

### DISCUSSION THREAD

**Discussion by ITstollman** [microsoft/AirSim#4787] [General]
I want to simulate the DJI mavic 3 enterprise (https://www.dji.com/mavic-3-enterprise) using ML algos.
In order to do so, I need to mimic that drone's spesific camera. 

Is there an option to ZOOM IN with the drone's camera in the simulation? ( as the real Mavic DJI can do)
Is there an option to have a camera that changes its point of view? ( as the real Mavic DJI can do)

Thanks for your help.
Itamar

**Comment by ppueyor** [comment]
Hi, u can explore the focal length of CinemAirSim: https://github.com/ppueyor/CinemAirSim

That has been already included in the main branch of AirSim.

Hope it helps!

**Reply by ITstollman** [reply]
Thank you very much.
It does help.
